src/doc/en/thematic_tutorials/tutorial-implementing-algebraic-structures.rst

```diff
@@ -319,7 +319,7 @@ Diagonal and Triangular Morphisms
 
 We now illustrate how to specify that a given morphism is diagonal or triangular
 with respect to some order on the basis, which means that the morphism is
-invertible and `Sage` is able to compute the inverse morphism automatically.
+invertible and Sage is able to compute the inverse morphism automatically.
```

Comment:
Should we say Sage or Sagemath ?

Reply:
There is a single occurrence of  `SageMath` throughout *all* the tutorials (and many occurrences of `Sage`), so I would stick with this or change it in a separate ticket.

Reply:
Thanks for the clarification. I have no preference.